designs/administrator/future-openshift/import-multiple-yaml/index.md

```diff
@@ -0,0 +1,38 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Import YAML for multiple resources
+This design details how users can import multiple resources using the **Import YAML** page.
+
+## Design
+
+The **Import YAML** page will still be accessible from the masthead. There are three ways a user can create multiple resources on this page. The instructions on the page should be updated to clarify how users can import YAML: *Drag and drop YAML or JSON files into the editor, or manually enter files and use `- - -` to separate each definition.*
+
+1) Users can manually enter multiple definitions in the editor, using `- - -` to delineate them.
+2) – Users can drag and drop a single YAML with multiple resources defined and separated with `- - -`.
+
+Both methods will show the following in the editor:
+
+![1](img/manual-entry.png)
+
+3) Users can drag and drop multiple files into the YAML area at once.
+
+![2](img/draganddrop.png)
+
+After the user clicks **Create**, the console will do a dry run of resource creation to find potential errors before beginning to actually create the resources. If errors are found, the user will be notified by an inline alert on the **Import YAML** page beneath the editor. If the dry run is completed without any issues, the resources will begin to be created and the user will be redirected to the **Importing resources** page.
```

Comment:
This is a general and ignorable comment but this interaction sort of uses both "Import" and "Create" throughout it to describe what the user is doing. I think its fine to use both because it does seem deliberate but it also almost seems like the button label could be "Import" or alternatively the Importing page could say "Creating resources..." Just a thought, definitely feel free to ignore this.

Reply:
That's a great point! I think I may have used Importing because the page is Import YAML but you truly are creating resources so I think you're right and I'll update the progress screen to be "Creating". Thanks!!